.browserslistrc

```diff
@@ -1,3 +1,4 @@
-last 3 versions
-not ie > 0
-not op_mini > 0
+> 0.5%
+last 2 versions
+Firefox ESR
```

Comment:
It covers only one version, do we really need this? It's not bad to have, I am just double-checking